crates/nargo/tests/test_data/global_consts/proofs/p.proof

```diff
@@ -1 +1 @@
\ No newline at end of file
+13329f6ec38c76baff11d3df961814a96f7e96607b9e2b8ccc1444eb71dd92c028bf9355c375b2d72d2fd2a213327aebaf4606ce66c6f6da1a1a2a1ee15bbcea05e8fded1056debc85f723cfb3c0b1099848be4e09e1f9a8ae886e72a2c1c2161c134d943b0dd57effbc637b9d12c91b430fc9bf40421f98ff37e181e0518830268ba3fd19e784671041e078c0afca24afc3df67c49f1008700b4d5d4805d0101195d00e5e49974271b91d2c028975e07572162ced267c000f5e22281162d8c11c3af50a6209f59f3936b620a7f76e2ba62855d439d2e8ab82ae8ce61e6764c7085a0a79947b8a233c68c90a7bd9b92ce60c0cae6f36ea48113cd248a913cc920d8cd7a928cb60e5ffc8b4b9c0d11628b587656f5b18e4277a279e766b12c5492b8704cc948e583d120bfcb5cfcb3b9a629363d0eeec80a96a8a63e603dea80e2382e19f5d4bf1d615c4cf2d634cf0c837c4c97f9ab98239de490de7b8b79aac12494f36e3e492a03c9f105783979cdde2028dafcda333f061d0c0f429b41a95108cc804e266841469b9e8ea5e9445068b9b42202b21c3a306a88bf0c95f19f821af71d5a90739507808a8076d6bc2809b49b95b34148b59f5d2baf95df1f6bb137414f769a89befba595c5d82f0df60a3f573d7d70d191ca8ab30742a80d477078281ddc8264959e21029181b297afcd95d967d03787aef2fc0ecc29adf59160450765bdf942c3a722750f0ddc31655259c424f5ef746da3e7cc725f6a92db9157463d0eee63e2979abfe9888d3a28c61c5ec2dfb8f8f2145ccfd876c173c3b240d70a31c2b49d3c871b26d6b0f6452553401440ff15678f9c38a086393568a075c2baaa40972b44e4ebb27a10af0541eaf744186ea38dab9b788cd6efb832c1ed874bfe21e5401f87b1ebec0712ad13e2ba97a4ca00968e2f18b59bf286abc28554a72d1bade9d5e9eebcc99652d806e62fff9f81c66b48bec216534e8f8c60dba6967a2d9d2e9b532bc756f15736cbe0877b6ae3c6a67fb7aca84abc72bbd155643b1d80d6bfa7e70f8c0e148d384a1442ea11632a7f8dcb96a23080935bb21f3c30ab353ad246d8f514cbb9993915d99875095d4506857c91864e7592567024f4733acd4e9d9d40de4552ba5825d21ee2d36e9133a3d54a047992e53a69525b535bcd1c92cff116405c2454ca0e8455f822a98c8253d9eead8fc872c82a7266adcd0045ef793e1c79b74443c42aefcbe2cdf74b6b332530b93c2be4a54e416834872f76c516dcb65d59578cfcae9d0b886eeceff2eebeecf88246822cc041a36ede1e4dc7110995db01a52a7c73e8956d40e01d4a6e2aa1fed878f3ff6391910f9098988add9919ac48b769335b2807930117de09fd05d05069ec3912a522c7e7ac475a97fc93818c0aadac77f299f2444e39fca06c9e1cc6230cafefa4b0c180d414ef2c84ef9ab1bed1bb5c39eaf1d795b713fd390abb4947c52d13ae708de7acf26ff77ba8f0ac2c0b4af3e1a647996e6472f8c0256408e59879eac4e14b23e140304516c35fa09f20751ebf0f3160ee3c8dddd09b98f16cac47ead7522ca537d080104225eda7826297aa1b10086b810eb07755ae7c74693cb8a22e22d453e6db2094fe45c9f4a31186c5790c359d2345b831421a4c8cb8b04ca30630df593d3e34ed5400b59bf2232372e8c7818146a44e7d52252b00c1ffdb9419f
```

Comment:
Some proof and Verifier file changes were committed, lets remove them. May also be a good time to add Verifier.tomls to the gitignore

Reply:
Should I remove the `Verifier.toml`'s all together right now if we are adding them to the gitignore? I can also just revert the changes to Verifier files for now as well

Reply:
I'd avoid removing them for now, it seems we still fail to prove if they're absent despite them being auto-generated now. E.g. proving fails with `cannot find input file located at ".../noir/crates/nargo/tests/test_data/6/Verifier.toml", run nargo build to generate the missing Prover and/or Verifier toml files` for me when the Verifier.toml is removed.

Reply:
OK, I just reverted some of the Verifier file changes that were committed